site-src/v1alpha2/faq.md

```diff
@@ -61,6 +61,8 @@
   guide what revisions are needed for the next release. It is possible
   that the next release will contain breaking changes.
 
+* **Q: What are the supported kubernetes version ?<br>**
+  A: Generally, we support Kubernetes 1.16+, but certain features like AppProtocol depend on Kubernetes 1.18 (opt-in) or 1.19 (on by default). There is not any other exceptions to the 1.16+ guideline right now.
```

Comment:
```suggestion
  A: Generally, we support Kubernetes 1.16+, but certain features like AppProtocol depend on Kubernetes 1.18 (opt-in) or 1.19 (on by default). There are not any other exceptions to the 1.16+ guideline right now.
```

Reply:
Do you mind wrapping these lines at 80 characters? Although it's not an absolute rule, we usually try to keep our line lengths under that length.